app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -142,13 +142,27 @@ message Auth {
 message CA {
   oneof ca {
     FileCA file_ca = 1;
+    EJBCA ejbca_ca = 2;
   }
 
   message FileCA {
     string cert_path = 1;
     string key_path = 2;
     string key_pass = 3;
   }
+
+  message EJBCA {
+    string server_url = 1;
+    // client cert authentication configuration
+    string key_path = 2;
```

Comment:
It would be great if you could add validations either here, using buf 

https://github.com/chainloop-dev/chainloop/blob/48faa88055f8377baecb8394f33b08e7c75607e8/app/controlplane/internal/conf/controlplane/config/v1/conf.proto#L157

or in the constructor on the ejbca, ideally in both.